src/Carbon/Carbon.php

```diff
@@ -2329,9 +2329,12 @@ public function isCurrentMonth()
     }
 
     /**
-     * Checks if the passed in date is in the same month as the instance month (and year if needed).
+     * Checks if the passed in date is in the same month as the instance´s month (and year if needed).
      *
-     * @param \Carbon\Carbon|\DateTimeInterface|null $date       The instance to compare with or null to use current day.
+     * The second parameter $ofSameYear will be changed to default to true in 2.x to be consistent with
```

Comment:
This is not an absolute certitude, it should be discussed further. Maybe we will rather split it into 2 distinct methods to better follow the principle of single responsibility. Or maybe it will be changed into a other way (native comparison, static method... PHPDoc comments does not need to detail future behavior. It should rather focus on current behavior.